### PR TITLE
Ports the Clown from RU

### PR DESCRIPTION
**Pull request by Blundir**
## About The Pull Request

Adds adminbuse job, the clown, i don't really think it needs further explanation.
Admins can open a job slot for it, any player can take the slot using latejoin menu. Cannot be accessed using any other means.
Additionally, ported the infamous waddling effect.
dreamseeker_MJJKqCZFXf

Complete with hud, minimap icons, custom radio prefixes and other.
dreamseeker_FE2BaLhqYT
GjvveK2yI8

Basically port of my stuff from RU.

## Why It's Good For The Game

HONK. More admin stuff is good, as well as good jokes.

## Changelog
:cl:
add: Added the Clown
/:cl:

### CODE REVIEW

File: code/datums/jobs/job/shipside.dm [hunk 1]
```diff
@@ -1223,3 +1223,52 @@ If you require any help, use <b>mentorhelp</b> to ask mentors about what you're
...
+		<b>Gamemode Availability</b>: Distress<br /><br /><br />
+		<b>Duty</b>: Your primary job is to HONK! Do pranks and funny jokes, tell anecdotes, annoy marines, try not to die, try to resist pepperball, survive, for the glory of the Honkmother! HONK!
+	"}
+
+	minimap_icon = "clown"
+
+
+/datum/job/terragov/civilian/clown/radio_help_message(mob/M)
+	. = ..()
+	to_chat(M, {"Your primary job is to uphold the fun, pranks, entertainment and good mood aboard the ship. Marines can get sad after a few weeks of cryosleep!"})
```
Comment (ivanmixo) [inline code/datums/jobs/job/shipside.dm:1259]:
```suggestion
	to_chat(M, "Your primary job is to uphold the fun, pranks, entertainment and good mood aboard the ship. Marines can get sad after a few weeks of cryosleep!")
```

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 2]
```diff
@@ -88,12 +88,21 @@
 	icon_state = "clown"
 	item_state = "clown"
 	slowdown = SHOES_SLOWDOWN + 1
-
+	var/enabled_waddle = TRUE
```
Comment (ivanmixo) [inline code/modules/clothing/shoes/miscellaneous.dm:91]: doc

**Review by ivanmixo — CHANGES_REQUESTED** [review]

**Comment by Wirpy** [comment]
speedmerge this now
